freeipa/src/main/java/com/sequenceiq/freeipa/service/stack/FreeIpaHealthDetailsService.java

```diff
@@ -55,7 +55,7 @@ public HealthDetailsFreeIpaResponse getHealthDetails(String environmentCrn, Stri
         HealthDetailsFreeIpaResponse response = new HealthDetailsFreeIpaResponse();
 
         for (InstanceMetaData instance: instances) {
-            if (instance.isAvailable()) {
+            if (instance.isAvailable() && !instance.isDeletedOnProvider()) {
```

Comment:
if it's available it can't be deleted on provider side, can it?

Reply:
You are correct.